README.md

```diff
@@ -37,6 +32,32 @@ When using the extension:
 	* The validation is XL type system-aware, as the extension includes type system information of a standard installation with the default plugins installed.
 * Embedded help is displayed when using code completion and when hovering over properties in the YAML document.
 
+## Updating the validation schema
+
+Please not that this only works as of XL Deploy and XL Release version 8.6.
```

Comment:
Instead: "As of XL Deploy and XL Release version 8.6, you can update the validation schema."